pkg/relayer/config/relayminer_configs_reader_test.go

```diff
@@ -37,9 +39,39 @@ func Test_ParseRelayMinerConfigs(t *testing.T) {
 
 			expectedError: nil,
 			expectedConfig: &config.RelayMinerConfig{
-				QueryNodeUrl:   &url.URL{Scheme: "tcp", Host: "localhost:26657"},
-				NetworkNodeUrl: &url.URL{Scheme: "tcp", Host: "127.0.0.1:36657"},
-				SigningKeyName: "servicer1",
+				QueryNodeGRPCUrl:       &url.URL{Scheme: "tcp", Host: "127.0.0.1:26658"},
+				NetworkNodeGRPCUrl:     &url.URL{Scheme: "tcp", Host: "127.0.0.1:36658"},
+				GRPCInsecure:           true,
+				PocketNodeWebsocketUrl: &url.URL{Scheme: "tcp", Host: "127.0.0.1:36657"},
```

Comment:
```suggestion
				PocketNodeWebsocketUrl: &url.URL{Scheme: "tcp", Host: "127.0.0.1:36657", Path: "websocket"},
```
Why is this not the same as the other config?